coordinator/provider/coordinator.go

```diff
@@ -782,29 +677,6 @@ func (qc *qdbCoordinator) DropKeyRange(ctx context.Context, id string) error {
 	return qc.db.DropKeyRange(ctx, id)
 }
 
-// TODO : unit tests
-func (qc *qdbCoordinator) DropShardingRule(ctx context.Context, id string) error {
-	// TODO: exclusive lock all routers
-	spqrlog.Zero.Debug().Msg("qdb coordinator dropping all sharding keys")
-
-	if err := qc.traverseRouters(ctx, func(cc *grpc.ClientConn) error {
-		cl := routerproto.NewShardingRulesServiceClient(cc)
-		dropResp, err := cl.DropShardingRules(ctx, &routerproto.DropShardingRuleRequest{
-			Id: []string{id},
-		})
-
-		spqrlog.Zero.Debug().
-			Interface("response", dropResp).
-			Msg("drop sharding rules response")
-		return err
-	}); err != nil {
-		return err
-	}
-
-	// Drop key range from qdb.
-	return qc.db.DropShardingRule(ctx, id)
-}
-
```

Comment:
Lets drop this typo: https://github.com/pg-sharding/spqr/pull/517/files#diff-03219affdd826d090e8c388844585a388392259b398e173ef6a72a53526bab28L877